translations/deckbuilding.tex/cs.po

```diff
@@ -561,16 +561,10 @@ msgstr "  \\begin{tikzpicture}\n"
 
 #. type: multicols
 #: sections/deckbuilding.tex:173
-#, fuzzy, no-wrap
-#| msgid ""
-#| "[overlay]\n"
-#| "  \\node[opacity=0.2] at (9, -3) {\\includegraphics[width=0.6\\linewidth]{\\art/cavalryman.png}};"
 msgid ""
-"  \\node[rotate=-28, opacity=0.15] at (0, 0) {\\includegraphics[width=0.7\\linewidth]{\\art/frost_ring.png}};\n"
+"  \\node[rotate=332, opacity=0.15] at (0, 0) {\\includegraphics[width=0.7\\linewidth]{\\art/frost_ring.png}};\n"
```

Comment:
```suggestion
"  \\node[rotate=\\string-28, opacity=0.15] at (0, 0) {\\includegraphics[width=0.7\\linewidth]{\\art/frost_ring.png}};\n"
```
There's a workaround for this in CS, both rotate and shift can accept negative values. See here: https://github.com/Heegu-sama/Homm3BG/pull/850/commits/ebfa24298b10bbabd7cc6f65602401b179c24f8e

Reply:
thanks ><